FeatureFiles/Language.feature

```diff
@@ -0,0 +1,32 @@
+﻿Feature: Language
+
+As a user I should be able to show what languages I know.
+ So that the people seeking for skills and languages can look at what details I hold.
+
+@regression
+Scenario: 1. Add new language to user profile
+	When User adds a new language and level to the profile section
+	Then The language is successfully added to the profile
+
+Scenario: 2. Edit existing language on user profile page
```

Comment:
you should think about state 

State is when one test does not depend on another for example test 1 adds language  English and test 2 updates the language English to French, then test 2 should not assume that  English is added by test 1 or any other language! Instead test 2 should first add the language it wants to update and then update it.  (Test 2 adds English and then updates it). This ensures that the test is in the correct state for the test to execute.